src/components/home-button/_home-button.scss

```diff
@@ -36,18 +36,19 @@ $includeHtml: false !default;
 
     &--brainly-plus &__logo-big {
       height: 41px;
+      width: 62px;
     }
 
     &--eodev &__logo-big {
-      width: 114px;
```

Comment:
i could not define width at all and base only on height (`width: auto`) but I noticed some issues on IE11 so I think it's more safe to do it this way.

Reply:
agreed 👍